cmodel.c

```diff
@@ -429,30 +434,31 @@ void boot(void) {
 
   IME = 1;
 
-  // wait for checksum verification bit
+  // wait for get checksum bit
   do {
     readCommand();
-  } while (!RAM_BIT_TEST(PIF_CMD_U, PIF_CMD_U_CHECKSUM));
+  } while (!RAM_BIT_TEST(PIF_CMD_U, PIF_CMD_U_GET_CHECKSUM));
 
   IME = 0;
 
-  memSwapRanges();
-  RAM_BIT_SET(PIF_CMD_U, PIF_CMD_U_CHECKSUM_ACK);
+  memSwapRanges();                        // copy PIF_CHECKSUM from external memory to internal memory
+  RAM_BIT_SET(PIF_CMD_U, PIF_CMD_U_ACK);  // ack that we received the checksum
 
   IME = 1;
 
-  // wait for clear pif ram bit
+  // wait for check checksum bit
   do {
     readCommand();
-  } while (!RAM_BIT_TEST(PIF_CMD_U, PIF_CMD_U_CLEAR));
+  } while (!RAM_BIT_TEST(PIF_CMD_U, PIF_CMD_U_CHECK_CHECKSUM));
 
   IME = 0;
 
   RAM(PIF_CMD_U) = 0;
   if (reset == 0)  // only run on cold boot, not on reset
     cicCompareInit();
 
-  // compare checksum
+  // compare checksum received by CPU to that received by PIF
```

Comment:
I think this would be clearer:
```suggestion
  // compare checksum received from CPU to that received from CIC
```